app/services/onestop_id.rb

```diff
@@ -56,68 +124,137 @@ def self.create_identifier(feed_onestop_id, entity_prefix, entity_id)
     ).to_s
   end
 
-  def self.validate_onestop_id_string(onestop_id, expected_entity_type: nil)
-    errors = []
-    is_a_valid_onestop_id = true
+  def self.validate_onestop_id_string(onestop_id)
+    klass = lookup(string: onestop_id)
+    return false, ['must not be empty'] if onestop_id.blank?
+    return false, ['invalid prefix'] unless klass
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.